### PR TITLE
PYTHON-4630 Add documentation for MongoClient.bulk_write

### PR DESCRIPTION
**Pull request by shruti-sridhar**
[PYTHON-4630](https://jira.mongodb.org/browse/PYTHON-4630)

### CODE REVIEW

**Review by blink1073 — APPROVED** [review]
Very nice!

File: pymongo/asynchronous/mongo_client.py [hunk 1]
```diff
@@ -2270,8 +2270,8 @@ async def bulk_write(
           1
```
Comment (ShaneHarvey) [inline pymongo/asynchronous/mongo_client.py:2270]: Could you link to the docs example page in this doc string? Like:
```
For more info see :doc:`/examples/client_bulk`.
```

**Review by ShaneHarvey — COMMENTED** [review]
Nice work!

**Review by ShaneHarvey — APPROVED** [review]
LGTM!

**Pull request merged**